query-feature-table/src/main/java/com/esri/arcgismaps/sample/queryfeaturetable/components/MapViewModel.kt

```diff
@@ -113,28 +124,13 @@ class MapViewModel(
                 // get the extent of the first feature in the result to zoom to
                 val envelope = feature.geometry?.extent
                     ?: return@launch messageDialogVM.showMessageDialog("Error retrieving geometry extent")
-                // update the map view to set the viewpoint to the state geometry
-                mapViewState.stateGeometry = envelope
+                // update the MapViewpointOperation to set the geometry of the returned feature
```

Comment:
"set the geometry of the returned feature" doesn't sound right to me here. Maybe the comment could be: 

```suggestion
                // update the viewpoint to the bounding geometry of the returned feature
```